Minimal example for a possible bug for combining `ipython_blocking` with JupyterLab:

* Run in notebook (works): [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/dokempf/ipython-blocking-mwe/main?filepath=mwe.ipynb)
* Run in Lab (does not work): [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/dokempf/ipython-blocking-mwe/main?filepath=..%2Flab%2Ftree%2Fmwe.ipynb)
